### PR TITLE
Edgility connector

### PR DESCRIPTION
**Pull request by atymic**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

### CODE REVIEW

**Review by vmanoharas — COMMENTED** [review]
Hello @atymic, 

Thank you very much for the PR. 

However, have you registered your connector through this link - if not, kindly register and create a PR. - https://forms.office.com/Pages/ResponsePage.aspx?id=v4j5cvGGr0GRqy180BHbR6Pz8V1FjqlLrUJHAW7-rtFUQk5DVEhBWTBDV05JRExVUUdGQTJTTVRaMy4u

Also kindly refer this documentation for connector certification process. https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-5-submit-your-connector-for-deployment

**Comment by atymic** [comment]
Application was submitted :)

**Comment by vmanoharas** [comment]
Thank you very much @atymic for submitting the request, let me approve the request, before review and approve this request.

**Comment by atymic** [comment]
Thanks @vmanoharas  :)

**Comment by atymic** [comment]
Got the email it's approved @vmanoharas 
Thanks!

File: certified-connectors/Edgility/apiDefinition.swagger.json [hunk 1]
```diff
@@ -0,0 +1,643 @@
...
+    "/v2/integrations/power-automate/subscribe": {
+      "x-ms-notification-content": {
+        "description": "Inbound Message",
+        "schema": {
+          "type": "object",
+          "properties": {
+            "event": {
+              "type": "string",
+              "description": "event",
+              "title": "",
```
Comment (vmanoharas) [inline certified-connectors/Edgility/apiDefinition.swagger.json]: please use a proper title

File: certified-connectors/Edgility/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,643 @@
...
+            "name": "Config",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "event": {
+                  "type": "string",
+                  "description": "event",
+                  "title": "",
```
Comment (vmanoharas) [inline certified-connectors/Edgility/apiDefinition.swagger.json]: please use a proper title

File: certified-connectors/Edgility/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,643 @@
...
+                    "value-title": "number"
+                  },
+                  "x-ms-visibility": "important"
+                },
+                "destination_url": {
+                  "type": "string",
+                  "description": "destination_url",
+                  "x-ms-notification-url": true,
+                  "x-ms-visibility": "internal",
+                  "title": ""
```
Comment (vmanoharas) [inline certified-connectors/Edgility/apiDefinition.swagger.json]: please use a proper title

File: certified-connectors/Edgility/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,643 @@
...
+              "properties": {
+                "data": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "ulid": {
+                        "type": "string",
+                        "description": "ULID",
+                        "title": "",
```
Comment (vmanoharas) [inline certified-connectors/Edgility/apiDefinition.swagger.json]: please use a proper title

File: certified-connectors/Edgility/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,643 @@
...
+                        "x-ms-visibility": "internal"
+                      },
+                      "name": {
+                        "type": "string",
+                        "description": "Name",
+                        "title": ""
```
Comment (vmanoharas) [inline certified-connectors/Edgility/apiDefinition.swagger.json]: please use a proper title

File: certified-connectors/Edgility/apiDefinition.swagger.json [hunk 6]
```diff
@@ -0,0 +1,643 @@
...
+                      },
+                      "status": {
+                        "type": "string",
+                        "description": "Status",
+                        "title": "",
```
Comment (vmanoharas) [inline certified-connectors/Edgility/apiDefinition.swagger.json]: please use a proper title

File: certified-connectors/Edgility/apiDefinition.swagger.json [hunk 7]
```diff
@@ -0,0 +1,643 @@
...
+              "type": "object",
+              "properties": {
+                "data": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "sender_id": {
+                        "type": "string",
+                        "description": "",
```
Comment (vmanoharas) [inline certified-connectors/Edgility/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/Edgility/apiDefinition.swagger.json [hunk 8]
```diff
@@ -0,0 +1,643 @@
...
+                        "title": "Sender ID",
+                        "x-ms-visibility": "important"
+                      },
+                      "label": {
+                        "type": "string",
+                        "description": "",
```
Comment (vmanoharas) [inline certified-connectors/Edgility/apiDefinition.swagger.json]: please use a proper description for the response. you can use something like "OK", "Sucess", "Bad request", etc.

File: certified-connectors/Edgility/apiDefinition.swagger.json [hunk 9]
```diff
@@ -0,0 +1,643 @@
...
+        "to": {
+          "description": "Destination phone number. Ideally in international format with the leading +, ie 61412345678. Local formatting will be parsed but is not recommended.",
+          "type": "string",
+          "example": "+61412345678"
+        },
+        "from": {
+          "description": "Sender ID, either a Dedicated Number, Verified Phone Number, Verified Alphanumeric Sender ID. When using numbers, use E164 format.",
+          "type": "string",
+          "example": "SHARED_NUMBER",
+          "title": "",
```
Comment (vmanoharas) [inline certified-connectors/Edgility/apiDefinition.swagger.json]: please use a proper title

File: certified-connectors/Edgility/apiDefinition.swagger.json [hunk 10]
```diff
@@ -0,0 +1,643 @@
...
+            "operationId": "ListSenderIds",
+            "value-path": "sender_id",
+            "value-collection": "data",
+            "value-title": "label"
+          }
+        },
+        "body": {
+          "description": "Message content. Unicode is accepted.",
+          "type": "string",
+          "title": ""
```
Comment (vmanoharas) [inline certified-connectors/Edgility/apiDefinition.swagger.json]: please use a proper title

**Review by vmanoharas — COMMENTED** [review]
Hello @atymic, 
Kindly resolve review comments?

**Comment by atymic** [comment]
@vmanoharas all comments addressed :)

**Review by vmanoharas — APPROVED** [review]
Hello @atymic 
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to Connector certification portal](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#update-submission-prior-to-certification) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in [Connector certification portal](https://connectorcertification.azurewebsites.net/) . Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in [Connector certification portal](https://connectorcertification.azurewebsites.net/), please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector.
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Comment by atymic** [comment]
Thanks @vmanoharas  :)

**Pull request merged**